Source/ROSIntegration/Classes/LookAtScanComponent.h

```diff
@@ -4,7 +4,7 @@
 #include <Components/ActorComponent.h>
 #include "RI/Topic.h"
 #include "ROSIntegrationGameInstance.h"
-#include "std_msgs/StdMsgsString.h"
+#include "std_msgs/String.h"
```

Comment:
What is the rationale behind that change?

Reply:
I believe that it is more consistent with the way ROS names its message, and also with the other message names in that folder.  It was the only one who had the "StdMsgs" prefix.

Reply:
Hi. That's true, but the change was introduced because some compilers had problems when including it. So i wouldn't change that right now if it isn't necessary.